trade-finance-manager-api/src/v1/controllers/deal.tasks.api-test.js

```diff
@@ -41,16 +41,52 @@ describe('createDealTasks', () => {
   });
 
   describe('shouldCreatePartiesTask', () => {
-    describe('when a deal has tfm.exporter.partyUrn', () => {
+    describe('when a deal has tfm.exporter.partyUrn but NO tfm.buyer.partyUrn and deal is BSS', () => {
+      it('should return true', () => {
+        const result = shouldCreatePartiesTask(mockDealWithPartyUrn);
+        expect(result).toEqual(true);
+      });
+    });
+
+    describe('when a deal does NOT have tfm.exporter.partyUrn but has tfm.buyer.partyUrn and deal is BSS', () => {
+      it('should return true', () => {
+        mockDealWithPartyUrn.tfm.parties.buyer = { partyUrn: '1234' };
+        mockDealWithPartyUrn.tfm.parties.exporter = '';
+        const result = shouldCreatePartiesTask(mockDealWithPartyUrn);
+        expect(result).toEqual(true);
+      });
+    });
+
+    describe('when a deal does NOT have tfm.exporter.partyUrn and NO tfm.buyer.partyUrn and deal is BSS', () => {
+      it('should return true', () => {
+        const result = shouldCreatePartiesTask(mockSubmittedDeal);
+        expect(result).toEqual(true);
+      });
```

Comment:
Ned test coverage with one as `1234` and another as `  ` please.

Reply:
done